collections-lists/ListExistsInAnotherCSharp/ListExistsInAnotherCSharp/CompareListsMethods.cs

```diff
@@ -0,0 +1,133 @@
+﻿using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Order;
+using System.Collections;
+using System.Numerics;
+using System.Runtime.InteropServices;
+
+namespace ListExistsInAnotherCSharp;
+
+[MemoryDiagnoser]
+[RankColumn]
+[Orderer(SummaryOrderPolicy.FastestToSlowest)]
+[GroupBenchmarksBy(BenchmarkLogicalGroupRule.ByParams)]
+public class CompareListsMethods
+{
+    private readonly Random _rand;
+
+    public CompareListsMethods()
+    {
+        _rand = new Random();
```

Comment:
No need for the constructor, just initialize the member field...although since we are not creating Random with a seed, we should just use Random.Shared